Tensile/TensileCreateLibrary.py

```diff
@@ -1059,6 +1061,70 @@ def verifyManifest(manifest) -> bool:
   
   return False if len(files) > 0  else True
 
+def parseArchitectures(inputArchs: str) -> Set[str]:
+  """Parse target architectures from format ``'arch1;arch2'`` to ``{'arch1', 'arch2'}``"
+  
+  Args:
+      inputArchs: Semicolon delimited gfx architectures passed via the ``--architecture`` option.
+
+  Returns:
+      Target architectures
+
+  Raises:
+      ValueError: If ``inputArchs`` (i) contains a comma or space, (ii) refers to an unknown Gfx 
+      architecture, or (ii) malformed or use incorrect delimiters.
+
+  """
+  inputArchs = inputArchs.lower()
+  if "," in inputArchs or " " in inputArchs:
+    raise ValueError("Commas and spaces are not supported as delimiters or in architecture strings. Check your input.")
```

Comment:
What about all other possible delimiters?

Reply:
The only other delimiters that I can imagine would be passed are `:` and `-`, both of which may appear in a Gfx architecture name. These cases still raise a value error later in the function when it can't match against a known Gfx name (since the string isn't split).

I have added unit tests to verify this behaviour.

Reply:
I guess the point I was trying to make is should we be doing this check? Does it buy us anything? If we document that semicolon or underscores are the only valid delimiters, do we need to check for all other possibilities? I am of the (not very strong) opinion we don't need to do this check.

Reply:
We don't _have_ to run this check, all it does it provide better information to the user for delimiters that may be used instead of `;` and `_`. I'm happy to remove this.